app/src/main/java/com/realityexpander/tasky/agenda_feature/presentation/agenda_screen/AgendaViewModel.kt

```diff
@@ -193,6 +195,14 @@ class AgendaViewModel @Inject constructor(
                 }
                 sendEvent(ShowProgressIndicator(false))
             }
+            is ClearErrorMessage -> {
+                _agendaState.update {
+                    it.copy(
+                        errorMessage = null
+                    )
+                }
+                sendEvent(ShowProgressIndicator(false))
```

Comment:
Why is it an event to show a progress indicator? I'd consider this a normal state

Reply:
OK I will refactor.